airtime_mvc/application/models/StreamSetting.php

```diff
@@ -464,7 +464,7 @@ public static function setDjLiveStreamPort($value)
 
     public static function getDjLiveStreamPort()
     {
-        return self::getValue("dj_live_stream_port", 8001);
+        return self::getValue("dj_live_stream_port", 8002);
```

Comment:
The Master port should be 8001 and the DJ port should be 8002 (or something different), since they can't be the same.